datadog_checks_base/requirements.in

```diff
@@ -33,6 +33,7 @@ requests_toolbelt==0.9.1
 requests-unixsocket==0.2.0
 simplejson==3.6.5
 six==1.16.0
+thefuzz==0.19.0
```

Comment:
We cannot ship this due to the license.

Instead try https://github.com/jamesturk/jellyfish, see https://jamesturk.github.io/jellyfish/functions/#jaro-winkler-similarity

Reply:
Thanks!

Reply:
Just wondering what types of licenses are allowed for use?

Reply:
Most, just stay away from (L)GPL and custom licenses